sdk/attestation/azure-security-attestation/src/main/java/com/azure/security/attestation/PolicyCertificatesAsyncClient.java

```diff
@@ -9,12 +9,12 @@
 import com.azure.core.annotation.ServiceMethod;
 import com.azure.core.http.rest.Response;
 import com.azure.security.attestation.implementation.PolicyCertificatesImpl;
-import com.azure.security.attestation.models.CloudErrorException;
-import com.azure.security.attestation.models.PolicyCertificatesModifyResponse;
-import com.azure.security.attestation.models.PolicyCertificatesResponse;
+import com.azure.security.attestation.implementation.models.CloudErrorException;
```

Comment:
Same as before, exchange types need to be public  - from non-implementation package.